### DISCUSSION TITLE
Release 2.2.1

### DISCUSSION THREAD

**Discussion by IsaacMarovitz** [Whisky-App/Whisky#657] [Announcements]
**Quality of Life:**
- Updates to Polish and French locales

**Bug Fixes:**
- Fixed excessive memory usage

**Full Changelog:** https://github.com/Whisky-App/Whisky/compare/v2.2.0...v2.2.1

<hr /><em>This discussion was created from the release <a href='https://github.com/Whisky-App/Whisky/releases/tag/v2.2.1'>Release 2.2.1</a>.</em>

**Comment by AkumetsuRin** [comment]
Hadn't played Cyberpunk for a while before today, and either it's the new version of Whisky, GPTK 1.1, or the 2.1 game patch, but with the same graphic settings, the frame rate decreased from 25-30 to 15-20 =/
